src/navability/graphql/DataBlobs.jl

```diff
@@ -70,6 +70,50 @@ mutation sdk_adddataentry(\$userId: ResourceId!, \$robotId: ResourceId!, \$sessi
 }
 """
 
+GQL_ADDBLOBENTRY = """
+mutation sdk_addblobentry(
+  \$userId: String!
```

Comment:
new system these will also be changing to `userName`, `robotName`, etc.  `robotId` is going to become a UUID, etc.

Reply:
It should be changed completely to this as far as I can tell:
```
addBlobEntry(
  blob: AddBlobEntryInput!
  options: AddBlobEntryOptionsInput
): Event
```

Reply:
yes input types look right for where we'd like to get to.  Not sure if the GQL schema has been updated to that everywhere yet.